### ISSUE TITLE
alt attributes

### ISSUE THREAD

**Issue by btrem** [microformats/microformats.io#53]
The alt attributes for the images are wrong. The alt attribute should be a textual *alternative* for the image, not a description. In particular, per https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt#icons_or_logos

> if an image is a corporate logo, the alt text should be the name of the company.
